src/display/annotation_helper.js

```diff
@@ -114,6 +118,90 @@ var AnnotationUtils = (function AnnotationUtilsClosure() {
     return container;
   }
 
+  function isBitSet(value, pos) {
+    // Note: pos is 1 based index
+    return !!(value & (1 << (pos - 1)));
+  }
+
+  function createWidgetAnnotationContainer(item) {
+    var element = document.createElement('div');
+    var width = item.rect[2] - item.rect[0];
+    var height = item.rect[3] - item.rect[1];
+    element.style.width = width + 'px';
+    element.style.height = height + 'px';
+    element.className = 'widgetContainer';
+
+    return element;
+  }
+
+  function createTextWidgetAnnotation(item, commonObjs) {
+    var isMultiline = isBitSet(item.fieldFlags, 13);
+    // var isPassword = isBitSet(item.fieldFlags, 14);
```

Comment:
Please remove commented out code if it is not used. It can be added in a follow-up patch when it becomes useful.